code/drasil-printers/Language/Drasil/TeX/Helpers.hs

```diff
@@ -157,7 +150,7 @@ item' bull = command1oD "item" (Just bull)
 maketitle, maketoc, newline, newpage, centering :: D
 maketitle = command0 "maketitle"
 maketoc   = command0 "tableofcontents"
-newline   = command0 "par" <> pure (text "~\n")
+newline   = command0 "par~\n"
```

Comment:
So is this now obsolete?

Reply:
The `newline` fix is in a separate PR, but I'll remove this one when the merge conflict between them arises. 😂